myosuite/tests/test_envs.py

```diff
@@ -22,9 +22,15 @@ def assert_close(prm1, prm2, atol=1e-05, rtol=1e-08):
         prm1_dict = flatten(prm1)
         prm2_dict = flatten(prm2)
         for key in prm1_dict.keys():
-            assert_close(prm1_dict[key], prm2_dict[key], atol=atol, rtol=rtol)
+            try:
+                assert_close(prm1_dict[key], prm2_dict[key], atol=atol, rtol=rtol)
```

Comment:
@elladyr -- this seems like a debug statement, right?

We need to check for them not try to circumvent these issues.

Reply:
I don't quite know where this came from. I have not worked on this file at all. Was it already like this before?

Reply:
reverted 91689968044638a33666ce48d07a99b86d781180